packages/splunkProcessor/tests/transformLogEvent.stepfunctions.test.js

```diff
@@ -38,15 +38,7 @@ describe("transformLogEvent tests for stepFunctions log groups", () => {
           field1: "foo",
           field2: "bar",
           details: {
-            input: JSON.stringify({
-              body: "foo",
-              headers: {
-                "apigw-request-id": "ac9ddfe0-030a-4bcd-b63b-d4d1a32381a0",
-                "X-Amzn-Trace-Id": "Root=1-6613feca-23ef52160d128597232a4c97",
-                "x-correlation-id": "b9156b99-13cc-4719-b832-73905e07e760",
-                "x-request-id": "a22cc83d-b40b-43a1-b5f7-71944cd498bc"
-              }
-            })
+            input: "redacted"
           },
           "apigw-request-id": "ac9ddfe0-030a-4bcd-b63b-d4d1a32381a0",
```

Comment:
Just checking that it's okay to have the IDs present here?

Reply:
ids need to present so we can correlate the logs together with ones from other systems.
Or if you mean having the actual ids in this file, then its fine as its just a file for unit tests